gravitee-am-repository/gravitee-am-repository-jdbc/src/main/java/io/gravitee/am/repository/jdbc/liquibase/UsernameUniquenessMigration.java

```diff
@@ -107,8 +107,7 @@ public SqlStatement[] generateStatements(Database database) throws CustomChangeE
 
                 if (!referenceUser.getSource().equalsIgnoreCase("gravitee") && !referenceUser.getSource().equalsIgnoreCase("cockpit")) {
                     logger.error("Organization Username '{}' migration only manages gravitee & cockpit identity providers",
-                            referenceUser.getUsername(),
-                            referenceUser.getSource());
```

Comment:
not sure how user's `source` here relates to kafka 😀 why is it removed?

Reply:
agreed, these two params need to be kept. Just adapt the log message to display the source at the end ("`Organization Username '{}' migration only manages gravitee & cockpit identity providers but source is {}`")